front/.gitignore

```diff
@@ -0,0 +1 @@
+public/js/app.js
```

Comment:
The ignore for the build artifact was wrong in previous commits, so I corrected it.
Also, "elm-stuff" should be a global ignore and included explicitly as needed (`!elm-stuff/`).

Reply:
@nqthqn Once you see this, please mark as resolved.

Reply:
I think to be friendly to newbie devs we should keep it as a local ignore.